packages/base/cards-grid.gts

```diff
@@ -19,6 +19,8 @@ import {
   cardTypeDisplayName,
 } from '@cardstack/runtime-common';
 import { tracked } from '@glimmer/tracking';
+// @ts-ignore no types
```

Comment:
This isn’t needed in `externals.ts`?

Reply:
yeah i was confused by that too--i only got a lint error here for the missing types